apps/e2e/lookup-storybook/src/app/country-field/country-field.component.html

```diff
@@ -2,7 +2,6 @@
   <sky-input-box labelText="Country field input" [disabled]="disabledFlag">
     <sky-country-field
       formControlName="countryControl"
-      [hideSelectedCountryFlag]="hideCountryFlag"
       [includePhoneInfo]="phoneInfoFlag"
     />
   </sky-input-box>
```

Comment:
in the e2e storybook there is a `hide-prepopulateed-flag` e2e test, can we delete that as well since it isnt testing anything anymore

Reply:
Removed